LibSerialize.lua

```diff
@@ -1265,10 +1279,13 @@ end
 
 function LibSerialize:SerializeEx(opts, ...)
```

Comment:
Won't there also be possible performance issues on deserialization? Should we add similar functionality there?